### ISSUE TITLE
Create tutorial for app?

### ISSUE THREAD

**Issue by datatalking** [LeMoussel/pydbhub#2]
Would it be appropriate or needed to create a tutorial for the app?

I'm thinking a walk-through of specific features that you list on the first page of the repo.

This was recommended by a friend so I am new to the DB app but have wanted to dig deeper into SQLite.

As a new contributor, I would like to help, are there documentation needs or low-level bugs that we need to address?

**Comment by justinclift** [comment]
Definitely.  Tutorials or anything that help people to use the application are definitely welcome by pretty much every OSS project. :smile:
